packages/generator/src/generator.ts

```diff
@@ -302,7 +302,7 @@ export abstract class Generator<
       return path.join(
         __dirname,
         "..",
-        process.env.NODE_ENV === "test" ? "./templates" : "./dist/templates",
+        process.env.NODE_ENV === "test" ? "./templates" : "./templates",
```

Comment:
I think it needs to be `dist/templates` when you're using it outside of monorepo.

Reply:
Oops ya, this is a mistake. Thanks for finding.